crates/rpc/src/executor.rs

```diff
@@ -189,16 +186,14 @@ pub(crate) fn map_broadcasted_transaction(
                     calldata: starknet_api::transaction::Calldata(std::sync::Arc::new(
                         tx.calldata.iter().map(|c| c.0.into_starkfelt()).collect(),
                     )),
-                    transaction_hash: starknet_api::transaction::TransactionHash(
-                        transaction_hash.0.into_starkfelt(),
-                    ),
-                    nonce: starknet_api::core::Nonce(Felt::ZERO.into_starkfelt()),
                 };
 
                 let tx = pathfinder_executor::Transaction::from_api(
                     starknet_api::transaction::Transaction::Invoke(
                         starknet_api::transaction::InvokeTransaction::V0(tx),
                     ),
+                    starknet_api::transaction::TransactionHash(transaction_hash.0.into_starkfelt()),
+                    None,
```

Comment:
What does this new `None` field do?

Reply:
That's `deployed_contract_address` -- we have it only for DEPLOY_ACCOUNT transactions.